astropy/io/fits/hdu/image.py

```diff
@@ -375,10 +375,11 @@ def _update_header_scale_info(self, dtype=None):
         # Added original_was_unsigned with the intent of facilitating the
         # special case of do_not_scale_image_data=True and uint=True
         # eventually.
-        if self._dtype_for_bitpix() is not None:
-            original_was_unsigned = self._dtype_for_bitpix().kind == 'u'
-        else:
-            original_was_unsigned = False
+        # FIXME: unused, maybe it should be useful?
```

Comment:
not sure about these, why keep it around?

Reply:
Because there are some bugs opened with this scaling stuff, and I already came at this and tried to use these lines ;)